ads/google/test/test-doubleclick.js

```diff
@@ -65,57 +49,14 @@ describes.sandboxed('writeAdScript', {}, env => {
   it('should use GPT and opt out of the GladeExperiment when' +
```

Comment:
References to "GladeExperiment" can be cleaned up in test cases.

Reply:
done